ui/components/component-library/modal-overlay/modal-overlay.scss

```diff
@@ -1,7 +1,8 @@
 .mm-modal-overlay {
-  position: absolute;
+  position: fixed;
   top: 0;
   left: 0;
   right: 0;
   bottom: 0;
+  z-index: $modal-z-index;
 }
```

Comment:
Fixing styling for the overlay so it displays properly inside the app against other elements with z-index.